app/main/views/briefs.py

```diff
@@ -174,19 +174,24 @@ def update_brief_status(brief_id, action):
     brief = Brief.query.filter(
         Brief.id == brief_id
     ).first_or_404()
-
-    if brief.framework.status != 'live':
-        abort(400, "Framework is not live")
+    framework_status = brief.framework.status
 
     action_to_status = {
         'publish': 'live',
         'withdraw': 'withdrawn',
         'cancel': 'cancelled',
         'unsuccessful': 'unsuccessful'
     }
-    if brief.status != action_to_status[action]:
-        previousStatus = brief.status
-        brief.status = action_to_status[action]
+    new_status = action_to_status[action]
```

Comment:
Much more readable 👍 But should we call it `new_brief_status` to differentiate between it and the `framework_status` (I'm easy either way)?

Reply:
Yeah, `old_brief_status`/`new_brief_status` would be a bit nicer.

Reply:
Done 👍